### PR TITLE
Section "Contributing"

### PR DESCRIPTION
**Pull request by mlep**
In response to https://github.com/JabRef/www.jabref.org/issues/27
Fell free to rework / add some sections

### CODE REVIEW

File: _sections/70-contribute.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+JabRef's help can sometimes be outdated or unsatisfactory. Please follow our [guidelines](https://github.com/JabRef/help.jabref.org/blob/gh-pages/CONTRIBUTING.md) to improve it.
+
+### Translation
+
+Although JabRef is currently available in 17 languages, some languages lack contributors.
+Hence, you may help in [translating JabRef interface](https://help.jabref.org/en/TranslatingGUI).
+
+### Money
+If you have no time or skills to contribute to any of the above, please, [make a donation](http://www.jabref.org/#donations).
```
Comment (Siedlerchr) [inline _sections/70-contribute.md]: Better: consider making a donation
Otherwise cgood

Comment (mlep) [inline _sections/70-contribute.md]: Changed!

**Comment by koppor** [comment]
I like the idea - should we remove http://help.jabref.org/en/FAQcontributing and 70-contribute.md from there? I don't want to force us to maintain two pages :)

**Comment by mlep** [comment]
These information should not change much through time, so the burden to maintain 2 pages is limited.
Currenlty, I have no idea what is the best entry point for potential contributors. Maybe wait to have some statistics about page consultation before deciding which one is the best?

**Comment by Siedlerchr** [comment]
We could check what Google Analytics says about it ;)

2016-10-17 16:03 GMT+02:00 MLEP <notifications@github.com>:

**Pull request merged**